backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/KeyspaceFactory.java

```diff
@@ -35,27 +35,39 @@
     private static final String SYSTEM_SCHEMA = "system_schema";
     private static final String KEYSPACES = "keyspaces";
     private static final String KEYSPACE_NAME = "keyspace_name";
+    private static final int CACHE_REPLICATION_FACTOR = 1;
 
     public static void createKeyspace(ClusterConfiguration clusterConfiguration, Cluster cluster) {
         if (clusterConfiguration.shouldCreateKeyspace()) {
             doCreateKeyspace(clusterConfiguration, cluster);
```

Comment:
maybe it's time to rename that method (and remove the do prefix at the same time)